src/Microsoft.Data.SqlClient/src/Interop/Windows/Interop.Libraries.cs

```diff
@@ -9,9 +9,8 @@ internal static partial class Libraries
         internal const string Kernel32 = "kernel32.dll";
         internal const string NtDll = "ntdll.dll";
 #if !NET8_0_OR_GREATER
```

Comment:
Why was this left in?

Reply:
The tests use that in multiple locations in SecurityWrapper.cs
https://github.com/dotnet/SqlClient/blob/4a9b070fee2a4267ef48a7052efa67af2eaf8ea5/src/Microsoft.Data.SqlClient/tests/tools/TDS/TDS.EndPoint/SSPI/SecurityWrapper.cs#L27